Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -1447,7 +1447,12 @@ void Connection::dispatchMessage(UniqueRef<Decoder> message)
     if (m_ignoreInvalidMessageForTesting)
         return;
 #endif
-    ASSERT(!didReceiveInvalidMessage);
+#if ASSERT_ENABLED
+    if (didReceiveInvalidMessage) {
+        WTFLogAlways("Received invalid message %s for destination %" PRIu64, description(message->messageName()).characters(), message->destinationID());
```

Comment:
I think there is log in `didReceiveInvalidMessage` functions, like `NetworkConnectionToWebProcess::didReceiveInvalidMessage`, so maybe we could just move `ASSERT(!didReceiveInvalidMessage);` to after `m_client->didReceiveInvalidMessage(...)`

Reply:
AuxiliaryProcessProxy calls RELEASE_LOG_FAULT which doesn't output to stderr like WTFLogAlways does